src/libpriv/rpmostree-jigdo-build.c

```diff
@@ -1059,12 +1059,12 @@ write_commit2jigdo (RpmOstreeCommit2JigdoContext *self,
  * into the OIRPM (what looks like a plain old RPM) by invoking `rpmbuild` using
  * a `.spec` file.
  *
- * The resulting "jigdo set" is then that OIRPM, plus the exact NEVRAs - we also
+ * The resulting "rojig set" is then that OIRPM, plus the exact NEVRAs - we also
```

Comment:
We might as well also `s/OIRPM/rojigRPM/` throughout when we rename the whole codebase too. WDYT? (Not for this pass, since as you mentioned, this is focusing on user-visible bits only).